utils/init-google-map.ts

```
@@ -53,7 +53,7 @@ const stylesByTheme = {
 
 let map: google.maps.Map, marker: google.maps.Marker, infoWindow: google.maps.InfoWindow;
 
-export const initGoogleMap = async (probe: Probe) => {
+export const initGoogleMap = async (probe: Probe, showPulse: boolean = false, showIW: boolean = true, verticalOffset: number | null = null) => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)